web/src/components/CategoriesContent.tsx

```diff
@@ -49,7 +57,17 @@ const CategoriesContent = ({ categories, categoryModel, t, ...props }: Categorie
         AfterContent={
           categoryModel.organization && <OrganizationContentInfo organization={categoryModel.organization} />
         }
-        Footer={categoryModel.embeddedOffers[0] && <EmbeddedOffer offer={categoryModel.embeddedOffers[0]} {...props} />}
+        Footer={
+          categoryModel.embeddedOffers[0] && (
+            <EmbeddedOffer
+              offer={categoryModel.embeddedOffers[0]}
+              city={city}
+              pathname={pathname}
+              cityCode={cityCode}
+              languageCode={languageCode}
+            />
+          )
```

Comment:
This is how it is supposed to be in the end, right?
```suggestion
          categoryModel.embeddedOffers.map(embeddedOffer => (
            <EmbeddedOffer
              offer={embeddedOffer}
              city={city}
              pathname={pathname}
              cityCode={cityCode}
              languageCode={languageCode}
            />
          ))
```

Reply:
Probably not. It's not quite clear how we handle multiple embeddedOffers yet. The idea would be to have on component which is responsible for multiple offers then. There could be multiple ways to implement this. E.g. The cms could handle requests to external (job offer) providers and provides an endpoint, which we call. Or we implement this in the frontend. Multiple embedded offers are not yet a thing, I fear.

Reply:
Okay, works for now. Thanks for explaining :)